style.css

```diff
@@ -0,0 +1,922 @@
+:root {
+  --font-family: 'Lato', sans-serif;
+}
+
+@media screen and (min-width: 768px) {
+  *,
+  *::after,
+  *::before {
+    list-style: none;
+    overflow-x: hidden;
+  }
+
+  body {
+    margin: 0;
+  }
+
+  .links {
+    background-color: #3e3e3e;
+    height: 30px;
+  }
+
+  nav ul {
+    display: flex;
+    font-family: var(--font-family);
+    font-size: 14px;
+    color: #272a31;
+    font-weight: 400;
+  }
+
+  .links ul {
+    display: flex;
+    font-size: 13px;
+    font-family: var(--font-family);
+    color: #fff;
+    margin-top: 0;
+    margin-bottom: -1rem;
+    padding-bottom: 0;
+    margin-right: 10rem;
+    position: relative;
+    float: right;
+  }
+
+  nav ul li {
+    margin: 0.5rem 1rem 0.5rem 1rem;
+  }
+
+  .links ul li {
+    margin: 0.3rem 1rem 0.2rem 1rem;
+  }
+
+  nav ul li:hover {
+    color: #ec5242;
+  }
+
+  .nav {
+    display: flex;
+    width: 100vw;
+    justify-content: space-around;
+    height: 75px;
+  }
+
+  footer strong {
+    font-size: 28px;
+    font-family: 'Lucida Sans', 'Lucida Sans Regular', 'Lucida Grande', 'Lucida Sans Unicode', Geneva, Verdana, sans-serif;
+    font-style: italic;
+  }
+
+  footer strong img {
+    height: 2.6rem;
+    margin-top: 0.5rem;
+  }
+
+  .nav strong a {
+    font-size: 28px;
+    font-family: 'Lucida Sans', 'Lucida Sans Regular', 'Lucida Grande', 'Lucida Sans Unicode', Geneva, Verdana, sans-serif;
+    font-style: italic;
+    color: #000;
+    text-decoration: none;
+  }
+
+  .feature img {
+    height: 10rem;
+    width: 9rem;
+  }
+
+  .red-img img {
+    height: 8rem;
+    width: 15rem;
+    margin: 0.5rem;
+  }
```

Comment:
- [ ] Kindly ensure that the two images with text are centered over them and also have a reddish background ☺️.

Your design 😎 
![image](https://user-images.githubusercontent.com/99394374/226080518-d2709b9c-f52a-490c-bc0e-b72360f9e5b5.png)

Original design 😎 
![image](https://user-images.githubusercontent.com/99394374/226080529-98450f2c-cf7e-4ac1-bac7-e7f0f2e4fd49.png)